server/github.go

```diff
@@ -22,16 +22,17 @@ const (
 
 func (s *Server) GetPullRequestFromGithub(ctx context.Context, pullRequest *github.PullRequest) (*model.PullRequest, error) {
 	pr := &model.PullRequest{
-		RepoOwner: *pullRequest.Base.Repo.Owner.Login,
-		RepoName:  *pullRequest.Base.Repo.Name,
-		Number:    *pullRequest.Number,
-		Username:  *pullRequest.User.Login,
-		FullName:  "",
-		Ref:       *pullRequest.Head.Ref,
-		Sha:       *pullRequest.Head.SHA,
-		State:     *pullRequest.State,
-		URL:       *pullRequest.URL,
-		CreatedAt: pullRequest.GetCreatedAt(),
+		RepoOwner:           *pullRequest.Base.Repo.Owner.Login,
+		RepoName:            *pullRequest.Base.Repo.Name,
+		Number:              *pullRequest.Number,
+		Username:            *pullRequest.User.Login,
+		FullName:            "",
+		Ref:                 *pullRequest.Head.Ref,
+		Sha:                 *pullRequest.Head.SHA,
+		State:               *pullRequest.State,
+		URL:                 *pullRequest.URL,
+		CreatedAt:           pullRequest.GetCreatedAt(),
+		MaintainerCanModify: pullRequest.GetMaintainerCanModify(),
```

Comment:
Commenting here because of GitHub limitations. But in L80, we are doing the same thing of logging the error. Let's propagate that upwards too.